db/migrations/20230208012216_products.sql

```diff
@@ -0,0 +1,15 @@
+-- migrate:up
+CREATE TABLE products (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    img_url VARCHAR(3000) NULL,
+    description VARCHAR(3000) NOT NULL,
+    main_ingredient VARCHAR(100) NOT NULL,
+    ingredient VARCHAR(3000) NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
```

Comment:
updated_at 컬럼은 데이터가 최초 생성 될 때엔 특정 시간이 기입되지 않도록 아래와 같이 수정해주세요.

```suggestion
    updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP
```